app/src/sharedTest/java/org/oppia/android/app/databinding/BUILD.bazel

```diff
@@ -27,6 +27,18 @@ _TEST_FILES = [
     for file in _TEST_FILES
 ]
 
+genrule(
+    name = "update_AppCompatCheckBoxBindingAdaptersTest",
+    srcs = ["AppCompatCheckBoxBindingAdaptersTest.kt"],
+    outs = ["AppCompatCheckBoxBindingAdaptersTest_updated.kt"],
+    cmd = """
+    cat $(SRCS) |
+    sed 's/import org.oppia.android.R/import org.oppia.android.app.databinding.R/g' |
+    sed 's/import org.oppia.android.databinding./import org.oppia.android.app.databinding.databinding./g' |
+    sed 's/import org.oppia.android.app.databinding.AppCompatCheckBoxBindingAdapters./import org.oppia.android.app.databinding.AppCompatCheckBoxBindingAdapters_updated./g' > $(OUTS)
```

Comment:
@Akshatkamboj14 this part is where the issue is coming from. I don't think we need to do the replacement for AppCompatCheckBoxBindingAdapters because it doesn't use any resources (so it isn't being regenerated).

Reply:
so @BenHenning How can we resolve this? should I remove line 38?